### ISSUE TITLE
BUG: vf-pattern variants or resources tab not working

### ISSUE THREAD

**Issue by sturobson** [visual-framework/vf-core#129]
steps taken to get to this:

```
git fetch origin
git reset --hard origin/develop
npm prune
npm install
gulp dev
```

navigate to the vf-summary pattern

expected behaviour:

clicking on a variant shows that variant
clicking on a resource tab shows that file

neither work

**Comment by sturobson** [comment]
actually resources 'kinda works' but not really

**Comment by khawkins98** [comment]
Ah! This is because my fix for multiple tabs in #123, I thought supported multiple tabs, but really seems to have only sorted nested tabs :(... 

Having a look.

**Comment by khawkins98** [comment]
Oh, wait. No, the JS works but this is because of how the Variants aren't client side tabs but rather links.

Just need to handle when there isn't a child container ... also loops back into how we shouldn't assume that a site wants to use our JS (#124)
